content/applications/sales/sales/invoicing/down_payment.rst

```diff
@@ -222,6 +224,70 @@ At this point, the flow is now complete.
    order to confirm the delivery. Otherwise, the delivered quantity can be entered manually directly
    on the sales order.
 
+.. _sales/invoicing/down_payment/100-percent-down-payments:
+
+Example: request 100% down payment
+==================================
+
+The process of requesting a 100% down payment is similar to the process of setting up a :ref:`50%
+down payment <sales/invoicing/down_payment/50-percent-down-payments>` but with fewer steps. The
+*Solar Panel Installation* product is being used in this example.
```

Comment:
```suggestion
down payment <sales/invoicing/down_payment/50-percent-down-payments>`, but with fewer steps. 

The following example focuses on a *Solar Panel Installation* product.
```
OPTIONAL: added comma and broke up sentences for enhanced sentence clarity and flow